File: app/src/main/java/com/evaluation/model/result/registered/Registered.java
package com.evaluation.model.result.registered;

import com.google.gson.annotations.SerializedName;

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
public class Registered {

    @SerializedName("date")
    private String mDate;

    @SerializedName("age")
    private Integer mAge;

    public String getDate() {
        return mDate;
    }

    public Integer getAge() {
        return mAge;
    }

}

File: app/src/main/java/com/evaluation/model/result/registered/Registered.kt
package com.evaluation.model.result.registered

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
data class Registered(
    @SerializedName("date")
    val mDate: String,

    @SerializedName("age")
    val mAge: Int
)